### ISSUE TITLE
Install Fails with Directus 10.8.3-10.9

### ISSUE THREAD

**Issue by BlackDahlia313** [WoLfulus/directus-extension-thumbhash#1]
Hey!

Was trying to update to 10.8.3 and than 10.9 and realized it was this extension causing my build to fail. Sharp was updated in Directus and the sharp dependency for this one is outdated. 

Could we get a update? I would manually but I don't wanna make a custom image on my desktop lol

**Comment by luigigorlero** [comment]
It might be a tougher job than just updating the dependancy, I was trying to fiddle around but got nowhere. If you use Docker, installing sharp does the trick

```
USER node
RUN pnpm add sharp
RUN pnpm install directus-extension-thumbhash@0.0.4

**Comment by davidpp** [comment]
https://github.com/WoLfulus/directus-extension-thumbhash/pull/2

**Comment by WoLfulus** [comment]
Solved in #2
